tests/nest_tests/resources/test_plasticity_dynamics_neuron.nestml

```diff
@@ -0,0 +1,45 @@
+"""
+test_plasticity_dynamics_neuron.nestml
+#######################################
+
+Copyright statement
++++++++++++++++++++
+
+This file is part of NEST.
+
+Copyright (C) 2004 The NEST Initiative
+
+NEST is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+NEST is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+"""
+model test_plasticity_dynamics_neuron:
```

Comment:
I'd insert an empty line above and add a comment describing the logic of the neuron: parrot on one port and a second port to support plasticity.

Reply:
Done!